### PR TITLE
Display time ago on block page

### LINKED ISSUE #1632: add time ago to block page
Regression, when looking at a block page it was previously possible to know how old a block was without having to do the math by hand to subtract the current time from the block time stamp. The functionality was lost in one of the page redesigns.

### PR DESCRIPTION
**Pull request by peterzen**
Fixes #1632

### CODE REVIEW

File: views/block.tmpl [hunk 1]
```diff
@@ -84,9 +84,9 @@
                 <div class="col-7 col-sm-8 text-left">
                     <span class="text-secondary fs13">Block Time</span>
                     <br>
-                    <span class="fs18 font-weight-bold lh1rem d-inline-block pt-1">{{.BlockTime.PrettyMDY}}</span>
+                    <span class="fs18 font-weight-bold lh1rem d-inline-block pt-1">{{.BlockTime.PrettyMDY}} <span class="fs13 text-secondary">{{.BlockTime.HMSTZ}}</span></span>
                     <br>
-                    <span class="fs14 text-secondary">{{.BlockTime.HMSTZ}}</span>
+                    <span class="text-secondary fs13"><span data-target="time.age" data-age="{{.BlockTime.UNIX}}"></span> ago</span>
```
Comment (chappjc) [inline views/block.tmpl]: Please add `jsonly` to the outer/fs13 span class list.

Comment (peterzen) [reply views/block.tmpl]: Done

**Comment by peterzen** [comment]
Yes, the intention was to free up some space for the relative time this
way.

Forgot to post a screenshot, sorry about that.

**Comment by buck54321** [comment]
This doesn't quite work. This breaks the whole layout pattern and doubles the width of the last element. What we've done elsewhere is to grab the element with javascript and change the hard date to a time since. This has the advantage that it doesn't change the layout, and those with JS disabled still get some data. Take a look at `time_controller.js` and elements with `data-target="time.age"`. 

There are other solutions too, but this doesn't work as is.

**Comment by peterzen** [comment]
@raedah do you have a screenshot of this page in the state before the regression?

**Comment by chappjc** [comment]
It was the major redesign, so it wouldn't be too helpful to see unfortunately.  How about putting the time below the date to keep the width down?

**Comment by peterzen** [comment]
> It was the major redesign, so it wouldn't be too helpful to see unfortunately. How about putting the time below the date to keep the width down?

How about this layout?
image

**Comment by chappjc** [comment]
That sits much better with me.  Lemme check out the code again.

**Comment by chappjc** [comment]
@buck54321 Does the revised layout work for you?

**Pull request merged**